lib/plugins/aws/package/compile/functions.js

```diff
@@ -216,6 +216,26 @@ class AwsCompileFunctions {
         artifactFilePath = path.join(this.serverless.serviceDir, '.serverless', artifactFileName);
       }
 
+      const runtimeManagement = this.provider.resolveFunctionRuntimeManagement(
+        functionObject.runtimeManagement
+      );
+      if (runtimeManagement.mode !== 'auto') {
+        const runtimeManagementMap = new Map([
```

Comment:
Let's not create it with every function call (define it in top level module context)